lib/hermetic/event_api.ex

```diff
@@ -61,6 +61,7 @@ defmodule Hermetic.EventAPI do
             extract_issue_ids(event["text"])
             |> Enum.map(&Attachment.new/1)
             |> Enum.reject(&is_nil/1)
+            |> Enum.take(3) # Don't spam more than 3 attachments
```

Comment:
Should be an option.

Reply:
Or is it fine to use a package constant like `@description_max_size`?

Reply:
I think OTP option would work best here (it's not some intrinsic constant, say unlike YouTrack's max int value), although any would document intent better.

I thought about something like:

```ex
  import ConfigMacro

  @doc ~S"""
  Max number of attachments to render.
  """
  @spec max_attachments() :: pos_integer()
  config :hermetic, max_attachments: 3
```

Then, you can just use it as `max_attachments()` and it will have a default (`3`). if one wants to, they can change it in `config.exs` via:

```ex
  config :hermetic, Hermetic.Attachment,
    max_attachments: 3
```

Reply:
done